CONTRIBUTING.md

```diff
@@ -156,6 +156,27 @@ apps/storybook/src/**Alert.test.tsx**
 
 - Contains the Cypress test for the component.
 
+### Importing icons
+
+#### In `packages/iTwinUI-react`:
+
+* Manually add the `<svg>` component to `utils/icons` and `utils/index.ts`
+* Import it from `utils`
+
+e.g.
+```tsx
+import { SvgClose, SvgInfoCircular} from '../utils';
+```
+
+#### In other modules:
+
+* Import it from `@itwin/itwinui-icons-react`
+
+e.g.
+```tsx
+import { SvgChevronRightDouble, SvgFolder } from '@itwin/itwinui-icons-react';
+```
+
```

Comment:
I don't think this is necessary. It doesn't matter what we do in other workspaces.

Reply:
**[DONE]** Sounds good, removed